api/sheets/places_in.md

```diff
@@ -8,37 +8,49 @@ grand_parent: API
 
 # Get Places Contained within Another Place
```

Comment:
Does it make sense to use Retrieve here? I also noticed that the sidebar has "Get ___" for all the functions except for this - would be good to standardize.

Reply:
We can do this, but I think the GET verb might be acceptable for the menu--at least partially because changing the menu to use `retrieve` would likely make the menu a little wide/unwieldy. I don't know though--what do you think?

Reply:
(As a starting point, I changed this header and made it sentence case as well.)

Reply:
Hm, yes, "retrieve" would make the side menu unwieldy. In fact, it looks like we have now removed "get" from all the API call links in the sidebar except for sheets. Perhaps the update should be to remove "Get" for sheets API calls?

Could we also standardize this by adding  "Retrieve" for the page title?

Reply:
sentence casing would be great as well!

Reply:
changed as requested :-)